import/import.js

```diff
@@ -1049,19 +1064,32 @@ async function processDocument(
   // If we're building this page, we can remove
   let builtFolderPath;
 
+  // If it's not archived and its not en-US dump it twice!
   if (isArchive || doc.locale !== "en-US") {
     builtFolderPath = Document.archive(
       getCleanedRenderedHTML(doc.rendered_html),
-      doc.html,
+      isArchive ? doc.html : null,
       meta,
       // The Document.archive() is used for genuinely archived content,
-      // but we also treat translated content the same way ultimately.
-      !isArchive
+      // but we also treat translated content the same way kinda.
+      !isArchive,
+      isArchive ? CONTENT_ARCHIVED_ROOT : CONTENT_TRANSLATED_RENDERED_ROOT
     );
+    builtFilePaths.add(path.join(builtFolderPath, "index.html"));
+
+    // If it's not archive, dump it a second time.
```

Comment:
Is a `!isArchive` check missing here?

Reply:
You were right. I fixed that in a second commit after.